lockable-object/src/main/java/com/iluwatar/lockableobject/App.java

```diff
@@ -0,0 +1,67 @@
+package com.iluwatar.lockableobject;
+
+import com.iluwatar.lockableobject.domain.Creature;
+import com.iluwatar.lockableobject.domain.Elf;
+import com.iluwatar.lockableobject.domain.Feind;
+import com.iluwatar.lockableobject.domain.Human;
+import com.iluwatar.lockableobject.domain.Orc;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The Lockable Object pattern is a concurrency pattern. Instead of using the "synchronized" word
+ * upon the methods to be synchronized, the object which implements the Lockable interface handles
+ * the request.
```

Comment:
Add here an explanation how this application implements the pattern. Please see the other patterns for an example.